[2025-03-12T09:28:09.069+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T09:28:09.134+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weekly_category_analysis.customers manual__2025-03-12T09:28:02.351344+00:00 [queued]>
[2025-03-12T09:28:09.163+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weekly_category_analysis.customers manual__2025-03-12T09:28:02.351344+00:00 [queued]>
[2025-03-12T09:28:09.165+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T09:28:09.217+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): customers> on 2025-03-12 09:28:02.351344+00:00
[2025-03-12T09:28:09.232+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weekly_category_analysis', 'customers', 'manual__2025-03-12T09:28:02.351344+00:00', '--job-id', '356', '--raw', '--subdir', 'DAGS_FOLDER/s3_data.py', '--cfg-path', '/tmp/tmpxcgyrdye']
[2025-03-12T09:28:09.229+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=240) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-12T09:28:09.235+0000] {standard_task_runner.py:105} INFO - Job 356: Subtask customers
[2025-03-12T09:28:09.236+0000] {standard_task_runner.py:72} INFO - Started process 248 to run task
[2025-03-12T09:28:09.410+0000] {task_command.py:467} INFO - Running <TaskInstance: weekly_category_analysis.customers manual__2025-03-12T09:28:02.351344+00:00 [running]> on host fc342882eb4f
[2025-03-12T09:28:09.616+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weekly_category_analysis' AIRFLOW_CTX_TASK_ID='customers' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T09:28:02.351344+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T09:28:02.351344+00:00'
[2025-03-12T09:28:09.619+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-12T09:28:09.619+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-12T09:28:09.620+0000] {logging_mixin.py:190} INFO - Current task name:customers state:running start_date:2025-03-12 09:28:09.139026+00:00
[2025-03-12T09:28:09.620+0000] {logging_mixin.py:190} INFO - Dag name:weekly_category_analysis and current dag run status:running
[2025-03-12T09:28:09.621+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T09:28:09.645+0000] {logging_mixin.py:190} INFO - Validation successful. Found 1445 customer records.
[2025-03-12T09:28:09.646+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-12T09:28:09.661+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T09:28:09.662+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weekly_category_analysis, task_id=customers, run_id=manual__2025-03-12T09:28:02.351344+00:00, execution_date=20250312T092802, start_date=20250312T092809, end_date=20250312T092809
[2025-03-12T09:28:09.712+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-12T09:28:09.713+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-12T09:28:09.714+0000] {logging_mixin.py:190} INFO - Dag name:weekly_category_analysis queued_at:2025-03-12 09:28:02.376103+00:00
[2025-03-12T09:28:09.715+0000] {logging_mixin.py:190} INFO - Task hostname:fc342882eb4f operator:PythonOperator
[2025-03-12T09:28:09.749+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T09:28:09.822+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T09:28:09.825+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
